javacord-core/src/main/java/org/javacord/core/entity/server/ServerImpl.java

```diff
@@ -1355,6 +1355,14 @@ public void selfUndeafen() {
                 this, getConnectedVoiceChannel(api.getYourself()).orElse(null), null, false);
     }
 
+    @Override
+    public CompletableFuture<Member> requestMember(long userId) {
+        return new RestRequest<Member>(getApi(), RestMethod.GET, RestEndpoint.SERVER_MEMBER)
+                .setUrlParameters(getIdAsString(), Long.toUnsignedString(userId))
+                .execute(result ->  new MemberImpl(api, this, result.getJsonBody(), null));
+
+    }
```

Comment:
```suggestion
    @Override
    public CompletableFuture<User> requestMember(long userId) {
        return new RestRequest<User>(getApi(), RestMethod.GET, RestEndpoint.SERVER_MEMBER)
                .setUrlParameters(getIdAsString(), Long.toUnsignedString(userId))
                .execute(result -> new MemberImpl(api, this, result.getJsonBody(), null).getUser());

    }
```

This way, we don't have to expose the member object (yet).